changelogs/unreleased/3803-disallow-multiline-strings-with-single-quotes.yml

```diff
@@ -0,0 +1,6 @@
+description: Trying to write a single quoted string over multiple lines now raises a syntax error.
```

Comment:
I find "single quoted" a bit of an ambiguous wording if we put this in the changelog but on the other hand I wouldn't know how to say this otherwise so no objections.

Reply:
How about :
```suggestion
description: Writing a string over multiple lines is now only supported for strings within triple quotes.
```

Reply:
I like it!